io.github.andreibachim.shortcut.yml

```diff
@@ -0,0 +1,24 @@
+app-id: io.github.andreibachim.shortcut
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+command: shortcut
+finish-args: [
+  '--socket=wayland',
+  '--socket=fallback-x11',
+  '--share=ipc',
+  '--device=dri',
+  "--filesystem=host",
+]
+modules:
+  - name: shortcut
+    buildsystem: simple
+    build-commands:
+      - install -D target/release/shortcut /app/bin/shortcut
```

Comment:
how is this build?

Reply:
Currently, it is built locally using [this script](https://github.com/andreibachim/shortcut/blob/main/scripts/release.sh). The plan is to move to github actions sometime in the near future.

Reply:
you should build it here on flathub. Like most are done.

Reply:
I have updated the manifest in order to build the binary in place.